components/script/dom/workers/workerglobalscope.rs

```diff
@@ -590,6 +594,15 @@ impl WorkerGlobalScope {
             return;
         }
 
+        let mut script = ScriptOrigin::external(
+            Rc::new(DOMString::from(script)),
+            self.worker_url.borrow().clone(),
+            ScriptFetchOptions::default_classic_script(global),
+            ScriptType::Classic,
+            global.unminified_js_dir(),
+        );
+        unminify_js(&mut script);
```

Comment:
I assume we don't want to always unminify the JavaScript?

Reply:
#34199